### ISSUE TITLE
ImageNet accuracy

### ISSUE THREAD

**Issue by ahyunSeo** [csuhan/ReDet#172]
Hello,

Thank you for your nice work!
I was trying to pre-train my own version of re-resnet on ImageNet.
Beforehand, I evaluated the ReResNet50-c8 model you provided with the metadata found in the following ('meta/val.txt')
https://github.com/open-mmlab/mmclassification/issues/850
where the val classes are assigned as 
ILSVRC2012_val_00000001.JPEG 65
ILSVRC2012_val_00000002.JPEG 970
ILSVRC2012_val_00000003.JPEG 230, ... .
I measured with the tools/test.py and I got only got top-1 0.1 and top-5 0.5.

Aren't the accuracies in your README evaluated in the validation set?
+ I checked the log and it seems like there are val accuracies.
Could you provide your meta files used in the following? 
https://github.com/csuhan/ReDet/blob/ReDet_mmcls/mmcls/datasets/imagenet.py#L32
https://github.com/csuhan/ReDet/blob/ReDet_mmcls/configs/_base_/datasets/imagenet_bs32.py#L33
If you used another meta data for training, please let me know.

Meanwhile, I feel that the file I used (download - https://github.com/BVLC/caffe/blob/master/data/ilsvrc12/get_ilsvrc_aux.sh)
matches with the training classes which are generated by sorting the foldernames as in https://github.com/csuhan/ReDet/blob/ReDet_mmcls/mmcls/datasets/imagenet.py#L74. 

For sanity check, I tested the following model provided by mmcls also and got top-1 76.28 and top-5 93.15.
https://mmclassification.readthedocs.io/en/latest/papers/resnet.html
    - [model(resnet50_8xb32)](https://download.openmmlab.com/mmclassification/v0/resnet/resnet50_8xb32_in1k_20210831-ea4938fc.pth)
    - [config(resnet50_8xb32)](https://github.com/open-mmlab/mmclassification/blob/master/configs/resnet/resnet50_8xb32_in1k.py)

Thanks for your answer in advance.

Best, 

Ahyun Seo

**Comment by ahyunSeo** [comment]
I was loading the wrong version of model.

I got 71.20/90.28 by the following:
The [raw](https://drive.google.com/file/d/1_d2igSp0wM8ypxTM1S14f5kCVjEyE6iI/view?usp=sharing) checkpoint is used to test the accuracy on ImageNet.

Sorry for the inconvenience!
